stories/utils.tsx

```diff
@@ -27,3 +27,30 @@ export const newLinterMessageUID = () => {
   _uid++;
   return `msg-${_uid}`;
 };
+
+export const rootAttributeParams = ({ fullscreenByDefault = false } = {}) => {
+  type Attribute = {
+    name: string;
+    value: string | null;
+  };
+
+  const fullscreen: Attribute = { name: 'Fullscreen', value: 'fullscreen' };
+  const normal: Attribute = { name: 'Normal', value: null };
```

Comment:
Could we get rid of this `normal` mode? It's confusing but this mode is exactly the same as without the add-on.

I'd also suggest putting a comment for:

> the main thing is that this story only makes sense when it's full screen, no padding

Reply:
The normal mode is required (otherwise all other stories will be fullscreen) but I found a better way to do it. I disabled the buttons for toggling between states and I think that makes it more clear.